packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx

```diff
@@ -102,6 +94,30 @@ export const CommentsSection: React.FC<CommentsSectionProps> = ({ disabled, vide
   const mdMatch = useMediaMatch('md')
   const placeholderItems = loading && !comments ? Array.from({ length: 4 }, () => ({ id: undefined })) : []
 
+  const handleCancelConfirmation = useCallback(
```

Comment:
This fn doesn't need to be in this component and be passed as a prop; we can just move it down to `CommentThread`